packages/core/src/electron-main/electron-main-menu.ts

```diff
@@ -0,0 +1,238 @@
+// *****************************************************************************
```

Comment:
Since this file doesn't use the logging utilities defined in the browser and backend, `console.debug` is just an alias of `console.log`, and all of the debug logs show up in the terminal. They should be removed or muted.

Reply:
Thanks for the remark. Sure. They won't be in the final version, at least as-is now.